CHANGES.md

```diff
@@ -18,6 +18,11 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 * Fix Telegraf migration when no containers present. (D2IQ-64507)
 
+* Marathon version bumped to 1.7.236
```

Comment:
The version number doesn't match what's in buildinfo.json?

Reply:
Copy pasta fail